### DISCUSSION TITLE
Question not an issue.

### DISCUSSION THREAD

**Discussion by Battler624** [quietvoid/dovi_tool#277] [Q&A]
Just wondering about the HDR10+ conversion from DV and how it differs from native HDR10+?

afaik they should technically be the same.

**Comment by quietvoid** [comment]
There's no way to convert Dolby Vision to HDR10+, so I don't know what you're asking about.
HDR10+ and Dolby Vision aren't the same.

**Comment by Battler624** [comment]
My apologies, I thought this tool would allow for a two-way transfer from DV to HDR10+ and from HDR10+ to DV.
